test/esinstall/node-env/node-env.test.js

```diff
@@ -1,35 +1,56 @@
+const fs = require('fs');
 const path = require('path');
-const {
-  existsPackageJson,
-  runTest,
-  testLockFile,
-  testWebModules,
-} = require('../esinstall-test-utils.js');
-
-require('jest-specific-snapshot'); // allows to call expect().toMatchSpecificSnapshot(filename, snapshotName)
+const {install} = require('../../../esinstall/lib');
```

Comment:
@matthewp I accidentally steered you wrong yesterday! I totally forgot that we are just using the Node API for `esinstall`, so no need to add it to `package.json` for this test or symlink it locally. We can just import the build directly. Sorry about that.

Reply:
Yeah this is actually how I figured out I wasn't rebuilding the TS. I'll update mine to do this too.